string state
string communication_status
string statusword
string driveflags
